lib/components/Card/index.js

```diff
@@ -72,6 +47,49 @@ CardContentStyled.defaultProps = {
   theme,
 };
 
+const CardStyled = styled.div`
+  // TODO Decide how to handle placing Card components within Grid/Box layouts.
+  // We'd like Cards to be able to stand on their own, but also be compatible
+  // within the Grid system. This margin-bottom rule, when placed within Grid
+  // results in extra spacing. Removing this rule will break current usage of
+  // this component.
```

Comment:
proposal: override the margin-bottom rule within `Box`:

CardStyled:

`margin-bottom: ${(props) => props.theme.units.fieldPadding};`

Box:

```
${Card} {
  margin-bottom: 0;
}
```